src/main/kotlin/org/jetbrains/research/ml/ast/util/FileUtil.kt

```diff
@@ -0,0 +1,33 @@
+package org.jetbrains.research.ml.ast.util
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.util.io.FileUtil
+import java.io.File
+
+fun getContentFromFile(filePath: String): String = File(filePath).readText(Charsets.UTF_8)
+
+fun createFolder(path: String) {
+    val file = File(path)
+    if (file.exists() && file.isFile) {
+        file.delete()
+    }
+    if (!file.exists()) {
+        file.mkdirs()
+    }
+}
+
+fun createFile(path: String, content: String = ""): File {
+    val file = File(path)
+    if (file.exists()) {
+        file.delete()
+    }
```

Comment:
If I understand correctly, `file.writeText(content)` rewrite the context of the file if it's not empty, so there is no need to delete the file, but I may be wrong

Reply:
you are right, thanks!